wallet/src/ui/app/pages/home/receipt/index.tsx

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import cl from 'classnames';
+import { useMemo, useEffect } from 'react';
+import { Navigate, useSearchParams, Link } from 'react-router-dom';
+
+import BottomMenuLayout, {
+    Content,
+    Menu,
+} from '_app/shared/bottom-menu-layout';
+import Icon, { SuiIcons } from '_components/icon';
+import ReceiptCard from '_components/receipt-card';
+import { useAppSelector, useAppDispatch } from '_hooks';
+import { getTransactionsByAddress } from '_redux/slices/txresults';
+
+import type { TxResultState } from '_redux/slices/txresults';
+
+import st from './ReceiptPage.module.scss';
+
+// Response pages for all transactions
+// use txDigest for the transaction result
+function ReceiptPage() {
+    const [searchParams] = useSearchParams();
+
+    // get tx results from url params
+    const txDigest = useMemo(
```

Comment:
These memos over search params are probably equally as costly as the underlying `get` call itself, so you really don't need them.

Reply:
Thanks done.